packages/changed-elements-react/src/widgets/comparisonJobWidget/components/VersionCompareSelectComponent.tsx

```diff
@@ -27,6 +28,9 @@ export interface VersionCompareSelectorProps {
 
   /** Named Versions to be displayed */
   namedVersions: CurrentNamedVersionAndNamedVersions | undefined;
+
+  /** Optional prop for a user supplied component to handle managing named versions.*/
+  manageNamedVersionProps?: ComponentProps<typeof ManageNamedVersions>;
```

Comment:
I would suggest to rename this prop to `manageNamedVersionsSlot` and make its type `ReactNode | undefined`.